lambdas/service/app.py

```diff
@@ -1379,7 +1379,22 @@ def generate_manifest(event, context):
                 more information refer to https://dss.data.humancellatlas.org
                 under `GET /files/{uuid}`. The only mandatory forwarded
                 parameter is `replica`, for which Azul only supports AWS.
-            ''')
+            '''),
+        params.query(
+            'version',
+            schema.optional(str),
+            description='''
+                A timestamp that generally follows
```

Comment:
The format of the timestamp is an implementation detail of the DSS, really. Clients should treat the version as an opaque string that is lexicographically sorted by time. We should only document that property. Can you check the DRS/DSS endpoints for this issue, too?